deploy/kicbase/Dockerfile

```diff
@@ -170,7 +171,29 @@ RUN install -m 0755 -d /etc/apt/keyrings \
     "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/debian \
     $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
     tee /etc/apt/sources.list.d/docker.list > /dev/null \
-    && clean-install docker-ce docker-ce-cli containerd.io docker-buildx-plugin
+    && clean-install docker-ce docker-ce-cli docker-buildx-plugin
+
+ # install containerd
```

Comment:
do we still need to install containerd if it is included in docker package? will it conflict docker if containerd version is not what docker wants?